rllib/tuned_examples/maml/pendulum-mass-maml.yaml

```diff
@@ -0,0 +1,26 @@
+pendulum-mass-maml:
+    env: ray.rllib.examples.env.pendulum_mass.PendulumMassEnv
+    run: MAML
+    stop:
```

Comment:
If these are actually learning, could you add some decent stopping criteria here? Like `episode_reward_mean` and `training_iteration` (or better `timesteps_total`).

Reply:
Above comment^^